v3/tutorials/11-kitties-workshop/b-kitties-frontend/index.mdx

```diff
@@ -416,7 +416,7 @@ Our `KittyCards.js` component will have three sections to it:
 
   See the screenshot below for reference:
 
-   ![kitty-transfer-shot.png](../../../../src/images/tutorials/11-kitties-workshop/kitty-transfer-shot.png)
+   ![kitty-transfer-shot.png](kitty-transfer-shot.png)
```

Comment:
so the convention you want is no hierarchy? it just discovers where an image us so long as it exists in `src/images`?

Reply:
I am just working with the requirement that was given to me.  As, in markdown, only provide the name of the file, not the tree.  
If we want to update this further, perhaps open it up for a discussion with the rest of the team.  Come up with either 
- Naming Convention, OR 
- Folder/Naming Convention

Let me know!

Reply:
As we will be migrating to new infrastructure based on `mkdocs` if we want to do this before in this stack already we should go with the planned structure schema.

@imadarai Should be moved to `/media/images/<section>/<category>` to mirror the content structure.
Eg.: `/media/images/docs/02-concepts/:image.png`

Depends on you @sacha-l @NukeManDan if you want to go through this already now, as you will be also changing the categories etc.. (maybe we just do only one level less before the migration `/media/images/<section>/` and add category sub-folders in new migration?)

Reply:
@lisa-parity should make the call on the structure she wants.
IMHO a verbose and complete internal dir path is better as it's more clear to everyone (including outside contributors) where things live, it's not "magicly" found by our rendering engine.